common/geth/client.go

```diff
@@ -59,6 +62,8 @@ func NewClient(config EthClientConfig, logger logging.Logger) (*EthClient, error
 			return nil, ErrCannotGetECDSAPubKey
 		}
 		accountAddress = crypto.PubkeyToAddress(*publicKeyECDSA)
+	} else {
```

Comment:
In the Batcher, it's the fireblock config precedent (and fallback to private key wallet); but here it's the privatekey precedent, should they match their precedence?